core/admin/mailu/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 class NoPingFilter(logging.Filter):
     def filter(self, record):
-        if (record.args['{host}i'] == 'localhost' and record.args['r'] == 'GET /ping HTTP/1.1'):
+        if (record.args['{host}i'] == '127.0.0.1' and record.args['r'] == 'GET /ping HTTP/1.1'):
```

Comment:
Maybe use the same test as the rspamd test?
`if (record.args['{host}i'] == 'localhost' and record.args['r'].endswitch( '/ping HTTP/1.1')):`

Reply:
Hmm, localhost wasn't working here that's why I was suggesting a change... but maybe it's misguided.